### PR TITLE
Adds back the missing stuff mentioned in #7

### PR DESCRIPTION
**Pull request by CortexPE**
## Introduction
Adds back the missing stuff mentioned in #7 
And makes the Logger a bit better.

## Tests
This was tested WORKING on my TestServer.

### CODE REVIEW

**Comment by CortexPE** [comment]
Also implements all the vanilla enchants... (I forgot to mention that)

File: src/pocketmine/Player.php [hunk 1]
```diff
@@ -3294,14 +3295,10 @@ public function handleItemFrameDropItem(ItemFrameDropItemPacket $packet) : bool{
 
 		$tile = $this->level->getTile($this->temporalVector->setComponents($packet->x, $packet->y, $packet->z));
 		if($tile instanceof ItemFrame){
-			$ev = new PlayerInteractEvent($this, $this->inventory->getItemInHand(), $tile->getBlock(), 5 - $tile->getBlock()->getDamage(), PlayerInteractEvent::LEFT_CLICK_BLOCK);
-			$this->server->getPluginManager()->callEvent($ev);
+			$this->server->getPluginManager()->callEvent($ev = new PlayerInteractEvent($this, $this->inventory->getItemInHand(), $tile->getBlock(), 5 - $tile->getBlock()->getDamage(), PlayerInteractEvent::LEFT_CLICK_BLOCK));
+            $this->server->getPluginManager()->callEvent($ev = new ItemFrameDropItemEvent($this, $tile->getBlock(), $tile, $tile->getItem()));
```
Comment (Matthww) [inline src/pocketmine/Player.php]: Spaces? You need to use tabs dude

**Comment by junkdude** [comment]
Please review for performance as well see if its a hacky method.

**Comment by ImagicalGamer** [comment]
Looks fine to me, add actual enchantments. Make sure they are implemented correctly, on EntityDamageEvent etc.

**Pull request closed without merging**